### PR TITLE
EE-675: refactored the project page layout

### PR DESCRIPTION
**Pull request by MaxWardle**

### CODE REVIEW

File: src/app/project/project-lg-md.component.scss [hunk 1]
```diff
@@ -0,0 +1,418 @@
...
+            color: white;
+        }
+
+    /*
+        Side Banner
+    */
+
+        .side-banner {
+
+            height: 100%; /* Full-height: remove this if you want "auto" height */
```
Comment (dhlevi) [inline src/app/project/project-lg-md.component.scss]: Looks good @MaxWardle!

I'm getting some squishing on the side banner on my end still, which causes the scrollbar to display. Looks like the "height: 100%" is overflowing a bit and adding a bunch of blank space below the map. It's good that the scrollbar shows up when this happens, but I wonder if just removing this to auto-size the panel will prevent it for most cases? Works on my end anyway.

File: src/app/project/project-lg-md.component.scss [hunk 2]
```diff
@@ -0,0 +1,418 @@
...
+                @media (min-width: 400px) {
+                    padding-right: 280px;
+                }
+            }
+            .map-container {
+                padding: 10px 0 10px 0;
+                display: block;
+                position: relative;
+                overflow: hidden;
+                // height: (400px - 33px); // to hide attribution
```
Comment (KitArmstrong) [inline src/app/project/project-lg-md.component.scss]: Remove if unused.

File: src/app/project/project-lg-md.component.scss [hunk 3]
```diff
@@ -0,0 +1,418 @@
...
+        }
+
+        /*
+            Contacts
+        */
+
+        .people {
+            // overflow: hidden;
+            background: #f7f7f7;
+            // color: #003366;
```
Comment (KitArmstrong) [inline src/app/project/project-lg-md.component.scss]: Remove.

File: src/app/project/project.component.ts [hunk 4]
```diff
@@ -113,6 +120,160 @@ export class ProjectComponent implements OnInit, OnDestroy {
...
+
+  // to avoid timing conflict with animations (resulting in small map tile at top left of page),
+  // ensure map component is visible in the DOM then update it; otherwise wait a bit...
+  // ref: https://github.com/Leaflet/Leaflet/issues/4835
+  // ref: https://stackoverflow.com/questions/19669786/check-if-element-is-visible-in-dom
+  private fixMap() {
+    if (this.elementRef.nativeElement.offsetParent) {
+      this.fitBounds(this.appFG.getBounds());
+
+      // option to centre the map instead of prov view
```
Comment (KitArmstrong) [inline src/app/project/project.component.ts]: I know this was just moved and not written by you, but can we clean up all the commented code while we are in here?

**Pull request merged**